src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -228,6 +228,18 @@ parameters:
     description: |
       Returns additional information about the category. The following optional fields can be requested:
       * count
+      * path
+    required: false
+    type: array
+    items:
+      type: string
+    collectionFormat: csv
+  categoryEntryIncludePathParam:
```

Comment:
It's a little confusing to have two of these, but we should fix it as part of https://alfresco.atlassian.net/browse/ACS-4923